### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD025.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#30] [General]
# https://blog.3place.cn/3place/fsd/FSD025.html

https://blog.3place.cn/3place/fsd/FSD025.html

**Comment by picklecai** [comment]
使她走向毁灭的是谁？是我还是你？

梅菲斯特这句话，叫人无可辩驳。当我很小的时候，也觉得冤枉。但现在，作为一个独立人格，就应该能认识到：独立的人，任何时候都不可以放弃自己的独立判断权利。

浮士德，虽然是个老人家。其实从故事的开始，他也相当于从幼年时期重新成长一遍。现在他的第一个教训就是：自己做的事，自己要担当，无法怪罪到别人身上，哪怕对方是魔鬼，哪怕对方引诱了你。

这就能呼应到，为什么梅菲斯特会特别重视契约和程序，非常有“法”的精神。

**Comment by picklecai** [comment]
我哪有天上地下所有的权柄？

谁都没有吧？越无知，越相信别的人（别的生灵）有。
